data/json/vehicleparts/roof.json

```diff
@@ -0,0 +1,45 @@
+[
+  {
+    "id": "roof_wood",
+    "type": "vehicle_part",
+    "name": "wooden roof",
+    "item": "frame_wood",
+    "color": "brown",
+    "durability": 130,
+    "breaks_into": [ { "item": "splinter", "count": [ 7, 9 ] } ],
+    "mounts": [ { "name": "on_roof", "volume": 20, "weight": 1000 } ],
```

Comment:
A volume restriction on a roof seems strange. Usually one put as much stuff there as one can tie down (respecting the weight limit).

Example: http://www.inspirational-quotes-short-funny-stuff.com/images/xsafe-driving-tips-funny-car-with-luggage.jpg.pagespeed.ic.sDuxBcgdSR.jpg